tests/integration/aliases/rename-component-and-child/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'application'
+}
+
+group 'com.pulumi.example'
+version '0.0.1-SNAPSHOT'
+
+var gprUser = project.findProperty("gpr.user")  ?: System.getenv("GPR_USER")
```

Comment:
We've removed everything connected with GPR_* this is not needed anymore and can be deleted! The build files on CI will find the Java SDK dependency via mavenLocal.

Reply:
done